### ISSUE TITLE
Email you have entered is not registered

### ISSUE THREAD

**Issue by renarj** [biemond/nl.zonneplan#1]
I tried to use this Homey app to get my ZonnePlan One inverter added, however it is not recognizing my account, it gives an error back "Email you have entered is not registered", am i doing something wrong?

**Comment by biemond** [comment]
Hey,

can you send me a diagnostic report or send me your email  ( biemond at gmail dot com ) .  or add my email to your setup. 

or maybe you use a different email on your setup.

thanks Edwin

**Comment by Frankovergoor** [comment]
Goedenavond Edwin,
I have the same problem. 
So I will also send you a mail as mentioned.
Kind regards,
Frank

**Comment by biemond** [comment]
yes. maybe it is some phone issue with javascript. I was mostly testing with the homey webapp on google chrome.

**Comment by biemond** [comment]
can we do this and see if we get some response 
<img width="691" alt="Screen Shot 2023-08-20 at 10 58 11" src="https://github.com/biemond/nl.zonneplan/assets/1085004/61f8328a-4ad0-46fc-bf48-d2ae2da7501f">


<img width="1650" alt="Screen Shot 2023-08-20 at 10 57 22" src="https://github.com/biemond/nl.zonneplan/assets/1085004/b0c04582-69f4-4f96-a754-3646f980a31f">

<img width="1616" alt="Screen Shot 2023-08-20 at 10 57 39" src="https://github.com/biemond/nl.zonneplan/assets/1085004/8e89ee22-e105-4c35-a6bb-3046d5311a1a">

**Comment by biemond** [comment]
thanks Frank.  can you try this https://homey.app/a/nl.zonneplan/test/

**Comment by Frankovergoor** [comment]
This one works. Thanks for teh great support.

**Comment by renarj** [comment]
I can confirm, the test version also resolves the issue for me

**Comment by biemond** [comment]
nice.  issue was I have node18 on homey 2023 and it seems fetch module is not there on homey versions with  node < 18 .

**Comment by biemond** [comment]
let me ask to publish this version.

**Comment by biemond** [comment]
oh it is live already. enjoy
